app/src/src_nonfreenet/org/breezyweather/sources/cwa/CwaResultConverter.kt

```diff
@@ -73,11 +72,14 @@ fun convert(
     id: String,
     ignoreFeatures: List<SecondaryWeatherSourceFeature>
 ): WeatherWrapper {
+    if (weatherResult.data == null) {
```

Comment:
Should be `weatherResult.data?.aqi?.getOrNull(0)?.town?.daily == null || weatherResult.data.aqi[0].town.hourly == null` (may need some `!!`), you're doing too many things after that can lead to Null Pointer Exceptions

Reply:
Done.